gdal raster neighbor: create as many output bands as input bands by default (#13165)
OSGeo/gdal: doc/source/programs/gdal_raster_neighbors.rst

      adding the sum of the (contributing, i.e. non nodata) weighted source pixels
      and dividing it by the sum of the coefficients in the kernel.

>     If :option:`--kernel` is specified several times, there will be as many
>     output bands as the number of values of the option times the number of bands
>     of the input dataset when :option:`--band` is not specified, or as many
>     output bands as the number of values of the :option:`--kernel` option.

Reviewer: ```suggestion
    If :option:`--kernel` is specified several times, there will be as many
    output bands as the number of values of the option times the number of input bands
    (selected using :option:`--band`).
```
